.travis.yml

```diff
@@ -4,7 +4,7 @@ services:
 otp_release:
   - 23.0
 elixir:
-  - 1.10.3
+  - 1.11.1
```

Comment:
@pedroperrone I think would be nice to have also to run the CI against elixir 1.3 to ensure backwards compatibility with this changes, as it is the first version supported https://github.com/drewolson/scrivener_ecto/blob/master/mix.exs#L8